### PR TITLE
📚 Update installation doc with proper link

### PR DESCRIPTION
**Pull request by delphiki**
# Proposed Changes

> Fix the link to set the IPv4 to static

## Related Issues

> N/A

[autolink-references]: https://help.github.com/articles/autolinked-references-and-urls/

### CODE REVIEW

**Comment by frenck** [comment]
CI fails on line length, could you take a look?

**Comment by delphiki** [comment]
It should be fixed :)

**Comment by delphiki** [comment]
Hey @frenck, do I still need to fix something?

**Comment by frenck** [comment]
yes, as I said before, the CI is failing. This hasn't been resolved.

CleanShot 2022-07-25 at 15 15 04

**Comment by delphiki** [comment]
> yes, as I said before, the CI is failing. This hasn't been resolved.
> 
> CleanShot 2022-07-25 at 15 15 04

I saw that, but I thought this was more of a CI problem, judging by the error: https://github.com/hassio-addons/addon-adguard-home/runs/7045382389?check_suite_focus=true

**Comment by frenck** [comment]
It isn't a CI issue. The files are not formatted with Prettier.
The CI tries to fix that but hasn't got permission to do so.

The solution is to format changed files using prettier.

**Comment by delphiki** [comment]
Okay I see, thanks for the explanation, I'll fix that asap :)

File: adguard/DOCS.md [hunk 1]
```diff
@@ -16,9 +16,10 @@ comparison to installing any other Home Assistant add-on.
 1. **Ensure your Home Assistant device has a
    [static IP and static external DNS servers!](https://github.com/home-assistant/hassos/blob/dev/Documentation/network.md#static-ip)**
    This is important! You **WILL** end up having issues if you skip this step.
-   - Change this setting in Supervisor:
-     [![Open your Home Assistant instance and show your Supervisor system information.](https://my.home-assistant.io/badges/supervisor_info.svg)](https://my.home-assistant.io/redirect/supervisor_info/)
-     (_Host → IP Address → Change → IPv4 → Static_)
+   - Change this setting in Network:
+    [![Open your Home Assistant instance and manage your systems network configuration.](https://my.home-assistant.io/badges/network.svg)](https://my.home-assistant.io/redirect/network/)
+     (_Settings → System → Network
+     → Configure network interfaces → Your Interface → IPv4 → Static_)
```
Comment (frenck) [inline adguard/DOCS.md:22]:
```suggestion
   - Change this setting in Network:
     [![Open your Home Assistant instance and manage your systems network configuration.](https://my.home-assistant.io/badges/network.svg)](https://my.home-assistant.io/redirect/network/)
     (_Settings → System → Network
     → Configure network interfaces → Your Interface → IPv4 → Static_)
```

**Review by frenck — APPROVED** [review]
Thanks, @delphiki 👍

**Pull request merged**